app/customer.py

```diff
@@ -0,0 +1,45 @@
+from app.shop import Shop
+from app.car import Car
+
+
+class Customer:
+    def __init__(self, dictionary: dict) -> None:
+        self.name = dictionary["name"]
+        self.product_cart = dictionary["product_cart"]
+        self.location = dictionary["location"]
+        self.money = dictionary["money"]
```

Comment:
it's preferable to use get() method instead of parenthesis everywhere for getting value by key to avoid KeyError